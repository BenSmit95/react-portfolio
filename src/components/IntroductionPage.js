import React from 'react';

const IntroductionPage = () => (
    <div>
        IntroductionPage component
    </div>
)

export default IntroductionPage