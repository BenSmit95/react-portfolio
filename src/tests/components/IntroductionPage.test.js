import React from 'react';
import { shallow } from 'enzyme';
import IntroductionPage from '../../components/IntroductionPage';

test('Should render IntroductionPage component correctly', () => {
    const wrapper = shallow(<IntroductionPage />);
    expect(wrapper).toMatchSnapshot();
});