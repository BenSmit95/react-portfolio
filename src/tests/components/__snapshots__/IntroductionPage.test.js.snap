// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render IntroductionPage component correctly 1`] = `
<div>
  IntroductionPage component
</div>
`;
